Client/Frontend/Home/HomePanelContextMenu.swift

```diff
@@ -10,19 +10,67 @@ protocol HomePanelContextMenu {
     func getContextMenuActions(for site: Site, with sourceView: UIView?, sectionType: FirefoxHomeSectionType) -> [PhotonRowActions]?
     func presentContextMenu(for site: Site, with sourceView: UIView?, sectionType: FirefoxHomeSectionType)
     func presentContextMenu(for site: Site, with sourceView: UIView?, sectionType: FirefoxHomeSectionType, completionHandler: @escaping () -> PhotonActionSheet?)
+
+    func getContextMenuActions(for highlightItem: HighlightItem, with sourceView: UIView?, sectionType: FirefoxHomeSectionType) -> [PhotonRowActions]?
+    func presentContextMenu(for highlightItem: HighlightItem, with sourceView: UIView?, sectionType: FirefoxHomeSectionType)
+    func presentContextMenu(for highlightItem: HighlightItem, with sourceView: UIView?, sectionType: FirefoxHomeSectionType, completionHandler: @escaping () -> PhotonActionSheet?)
+
 }
 
 extension HomePanelContextMenu {
+    // MARK: Site
     func presentContextMenu(for site: Site, with sourceView: UIView?, sectionType: FirefoxHomeSectionType) {
         presentContextMenu(for: site, with: sourceView, sectionType: sectionType, completionHandler: {
             return self.contextMenu(for: site, with: sourceView, sectionType: sectionType)
         })
     }
 
     func contextMenu(for site: Site, with sourceView: UIView?, sectionType: FirefoxHomeSectionType) -> PhotonActionSheet? {
-        guard let actions = getContextMenuActions(for: site, with: sourceView, sectionType: sectionType) else { return nil }
+        guard let actions = getContextMenuActions(for: site,
+                                                  with: sourceView,
+                                                  sectionType: sectionType)
+        else { return nil }
+
+        let viewModel = PhotonActionSheetViewModel(actions: [actions],
+                                                   site: site,
+                                                   modalStyle: .overFullScreen)
+        let contextMenu = PhotonActionSheet(viewModel: viewModel)
+        contextMenu.modalTransitionStyle = .crossDissolve
+
+        let generator = UIImpactFeedbackGenerator(style: .heavy)
+        generator.impactOccurred()
```

Comment:
Haptics feedback is new, we need this on all context menu?

Reply:
This was copied over from existing function, and is current behaviour on `main`.